### ISSUE TITLE
Commands are missing in reader mode

### ISSUE THREAD

**Issue by konhi** [marcusolsson/obsidian-vale#12]
I just discovered this plugin and it seems amazing. Thanks for your incredible work, it's absolutely impressive. Now, why would anyone use Grammarly, when we have this! 😄

---
I wanted to check if I did any mistakes. It surprised me that the vale commands were missing from command pallete. I thought that the plugin broke, so I tried configuring it, installing a non-managed version of CLI, and yet it still didn't work. Then, I realized that this plugin only allows to check document when user is in source mode - which might be unintuitive for... some users! 😅

**TL;DR: please consider allowing user to check while in reader mode.**
